roles/tftp-server/files/d-i/late_command.sh

```diff
@@ -40,7 +39,8 @@ if [ ! -z ${inventory_repo} ]; then
     fi
 fi
 
-cat > /usr/local/sbin/ansible-up <<EOF
+app=/usr/local/sbin/ansible-up
```

Comment:
I don't think people usually refer to scripts as apps.

Reply:
am I not a people? ;)

fixed:
 42 script=/usr/local/sbin/ansible-up
 43 cat > $script <<EOF